### ISSUE TITLE
[Issue] `v0.5.2`  breaks `regctl image inspect --format {{}}`

### ISSUE THREAD

**Issue by felipecrs** [regclient/regclient#551]
```console
❯ docker run --rm regclient/regctl:v0.5.1 image inspect regclient/regctl:latest --format '{{ jsonPretty .Config.Labels }}'
{
  "maintainer": "",
  "org.opencontainers.image.authors": "Regclient contributors",
  "org.opencontainers.image.created": "2023-09-27T19:49:08Z",
  "org.opencontainers.image.description": "",
  "org.opencontainers.image.documentation": "https://github.com/regclient/regclient",
  "org.opencontainers.image.licenses": "Apache 2.0",
  "org.opencontainers.image.revision": "56e0d20d046938b4bd9f24a780c4c6cdc2a4cfdf",
  "org.opencontainers.image.source": "https://github.com/regclient/regclient.git",
  "org.opencontainers.image.title": "regctl",
  "org.opencontainers.image.url": "https://github.com/regclient/regclient",
  "org.opencontainers.image.vendor": "",
  "org.opencontainers.image.version": "v0.5.2"
}

❯ docker run --rm regclient/regctl:v0.5.2 image inspect regclient/regctl:latest --format '{{ jsonPretty .Config.Labels }}'
template: out:1:21: executing "out" at <.Config.Labels>: can't evaluate field Config in type *blob.BOCIConfig
```

**Comment by sudo-bmitch** [comment]
Thanks. Looks like some of the refactoring of the blob package broke things. I'll try to get this fixed over the weekend.

**Comment by sudo-bmitch** [comment]
If the fix to the edge image gives you any trouble, let me know and we can reopen this. And let me know if you need a release for your workflow. Normally I'd do one in a month, but with a regression, I'm happy to push one after a week (to give any other issues a chance to show up).

**Comment by felipecrs** [comment]
1. Works fine using the edge image.
1. No rush from my side to release a new version: my workflows currently pins to 0.5.1.

Thanks a lot!

**Comment by sudo-bmitch** [comment]
Thanks again for the report! The fix for this is now released in v0.5.3. If it gives you any trouble, don't hesitate to reach out.

https://github.com/regclient/regclient/releases/tag/v0.5.3
